packages/x-date-pickers/src/internals/models/props/toolbar.ts

```diff
@@ -3,6 +3,11 @@ import { DateOrTimeView } from '../views';
 
 export interface BaseToolbarProps<TValue, TView extends DateOrTimeView>
   extends ExportedBaseToolbarProps {
+  /**
+   * If `true`, show the toolbar even in desktop mode.
```

Comment:
Not sure which one is valid, we have both in our descriptions

```suggestion
   * If `true`, shows the toolbar even in desktop mode.
```

Reply:
Do we? 🤔 I couldn't find any usages...
To me `show` sounds a bit more correct in this context.

Reply:
> If `true`, renders as disabled.
> If `true`, calls `renderLoading` instead of rendering the day calendar.

> If `true`, disable values after the current date for date components, time for time components and both for date time components.
> If `true`, force firing the `onChange` callback


I guess it depends if we consider it do be a short notation of `the component shows` or if it's an infinite.

The core also uses both

> If `true`, expands the accordion, otherwise collapse it.

> If `true`, disables gutter padding.


> If `true`, display the list in horizontal direction.

> If `true`, hide the selected options from the list box.

Sometime it even have the explicit `it`

> If `true`, it removes the margin between two expanded accordion items and the increase of height.

They also have a few sentences with `will`

> If `true`, will allow focus on disabled items.

____

In summary, nobody agrees :laughing:

Reply:
> In summary, nobody agrees 😆

Good summary. 😆 
I think that we could leave this case as is for now.
But maybe @samuelsycamore has an idea of which form would be the most correct? We could unify such cases in the future if we decide on an approach. 🤔

Reply:
Clearly, let's not block this PR for it
I was thinking out loud :laughing: